src/services/api/transactionsListService.js

```diff
@@ -0,0 +1,21 @@
+import axios from "axios";
+
+const baseUrl = "http://localhost:8000/transactions";
+
+export const requestForTransactionsList = ({ actualPage, entriesOnPage }) => {
```

Comment:
is not request, it's just a get call for transactions list
so name it `getTransactionsList`

Reply:
`entriesOnPage` => `transactionsListSize`/`transactionsListLength`/`listLength`

Reply:
`actualPage` => `currentPage`